lib/formtools/renderers-cell.js

```diff
@@ -156,7 +156,16 @@ var documentArrayRenderer = module.exports.documentArray = function documentArra
 
     });
 
-}
+};
+
+var textRenderer = module.exports.text = function textRenderer(val, record, fieldName, model, callback) {
+
+    if (val === undefined) {
```

Comment:
The best way to check for undefined in this instance is to use `typeof val === 'undefined'` for reasons describe in this thread https://stackoverflow.com/questions/4725603/variable-undefined-vs-typeof-variable-undefined